modules/integration/pages/ledger-quick-start.adoc

```diff
@@ -1,54 +1,54 @@
 
-== Basic terminology to get you started
+== Basic quickstart integration terminology 
 
 The Internet Computer is primarily a distributed and decentralized platform for running software. 
-When developers write applications that run on the Internet Computer, they deploy their programs in the form of a conceptual computational unit called a **canister**. 
-A canister is similar to a “smart contract” in that it consists of both program code and state running, replicated, on a block chain network with security and liveness guarantees.
+When you write applications that run on the Internet Computer, you deploy your programs in the form of a conceptual computational unit called a **canister**. 
+A canister is similar to a "smart contract" in that it consists of the source code of a program as well as its running state, and is replicated on a blockchain network that guarantees security as well as liveness.
 
-End-users—or other canisters—can send messages to canister functions to perform specific operations. 
-The messages can be either **query calls** that retrieve information without saving state or **update calls** that change and preserve state.
+End-users or other canisters can send messages to canister functions to perform specific operations. 
+The messages can be either **query calls** that retrieve information without saving the state of application data or **update calls** that change and preserve the state.
 The order in which updates are executed is agreed upon by using a consensus protocol between all {IC} nodes that run the canister.
 
 == Ledger canister overview
 
 The Internet Computer Protocol (ICP) implements token management using a specialized canister, called the **ledger canister**.
-There is a single ledger canister which runs alongside other canisters on the Internet Computer. 
-The ledger canister is, in essence, a smart contract that holds **accounts** and **transactions**.
+There is a single ledger canister which runs alongside all other canisters on the Internet Computer. 
+The ledger canister is basically a smart contract that holds **accounts** and **transactions**.
 These transactions either **mint ICP tokens** for accounts,  **transfer ICP tokens** from one account to another, or **burn ICP tokens**, eliminating them from existence.
-The ledger canister maintains a traceable history of all transactions starting from the genesis state.
+The ledger canister maintains a traceable history of all transactions starting from its genesis, or beginning, state.
 
 === Accounts
 
-An account belongs to (and is controlled by) an account owner (which must be an IC principal); no account can be owned by two or more IC principals (i.e. no “joint accounts”).
+An account belongs to and is controlled by the account owner who must be identified by a unique IC principal. No account can be owned by two or more IC principals (i.e. no "joint accounts").
```

Comment:
I think the previous wording was correct: the principal is an entity who can be authenticated by the IC. So the owner cannot be identified by an IC principal, but rather, then owner is an IC principal.

Reply:
You've got it. Thanks!